airflow/providers/delta/sharing/example_dags/example_delta_sharing.py

```diff
@@ -0,0 +1,77 @@
+#
```

Comment:
General note - I see you've used new system test design (good work!) but I also see example dag here. One of the purposes of new design of system tests was to have system tests that are used as examples - that's why it's better to keep only the one in tests/system and link documentation (like ``howto_delta_sharing_operator`` tag) to system tests. That way you're ensuring that your examples are actually executed and working.

Reply:
yes, forgot to remove it - will do it when addressing other comments.
thanks